content/go/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,150 @@
+---
+Title: 'Data Types'
+Description: 'Classifications of data used in everyday programming. In Go, there are seven fundamental data types: Numbers, Strings, Booleans, Arrays, Structs, Slices, and Pointers.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Go'
+CatalogContent:
+  - 'introduction-to-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+Data types are the classifications given to the different kinds of values used in everyday programming. In Go, there are seven fundamental data types:
+
+- [Numbers](#Numbers)
+- [Strings](#Strings)
+- [Booleans](#Booleans)
+- [Arrays](#Arrays)
+- [Structs](#Structs)
+- [Slices](#Slices)
+- [Pointers](#Pointers)
+
+## Number
+
+In Go, any number is regarded as number, but there are three sub-categories:
+
+- Integers (signed and unsigned)
+- Floating-point numbers
+- Complex numbers
+
+### Integers
+
+Integers are number-types that are either:
+
+- Signed (`int`) integers that can be positive or negative.
+- Unsigned (`uint`) integers ranging from 0 to positive infinity.
+
+Each integer value comes in 4 different memory sizes.
+
+- 8-bit: `int8 / uint8`
+- 16-bit: `int16 / uint16`
+- 32-bit: `int32 / uint32`
+- 64-bit: `int64 / uint64`
+- 32- or 64-bit based on system architecture: `int / uint`
+- synonym for int32: `rune`
+- synonym for int8: `byte`
+
+### Floating-Point Numbers
+
+floating-point numbers can contain a decimal point. There are two different sizes.
+
+- 32-bit: `float32`
+- 64-bit: `float64`
+
+### Complex Numbers
+
+Complex numbers are composed of a real number and an imaginary number. The real number is either a float. There are two different sizes:
+
+- 32-bit float + imaginary number: `complex64`
+- 64-bit float + imaginary number: `complex128`
+
+```go
+  // initialize with complex() taking 2 arguments: 1) a float and 2) a complex number
```

Comment:
Capitalize first letter in the comment, take out the numbers, and take out all the unnecessary spaces:

```suggestion
// Initialize with complex() taking 2 arguments: a float and a complex number
```